ui/app/AppLayouts/Profile/Sections/NotificationsContainer.qml

```diff
@@ -12,7 +12,72 @@ ScrollView {
     contentHeight: notificationsContainer.height
     clip: true
 
-    Rectangle {
+    ModalPopup {
+        id: mutedChatsModal
+        title: qsTr("Muted chats")
+        property bool showMutedContacts: false
+
+        ListView {
+            id: mutedChatsList
+            anchors.top: parent.top
+            visible: true
+            height: childrenRect.height
+            anchors.left: parent.left
+            anchors.right: parent.right
+            model: mutedChatsModal.showMutedContacts ? profileModel.mutedContacts : profileModel.mutedChats
+            delegate: Rectangle {
+                anchors.left: parent.left
+                anchors.right: parent.right
+                height: contactImage.height + Style.current.smallPadding * 2
+                color: Style.current.transparent
+
+                StatusIdenticon {
+                    id: contactImage
+                    height: 40
+                    width: 40
+                    chatName: model.name
+                    chatType: model.chatType
+                    identicon: model.identicon
+                    anchors.left: parent.left
+                    anchors.verticalCenter: parent.verticalCenter
+                }
+
+                StyledText {
+                    id: contactInfo
+                    text: model.chatType !== Constants.chatTypePublic ?
+                        Emoji.parse(Utils.removeStatusEns(Utils.filterXSS(model.name)), "26x26") :
```

Comment:
We can render emojis in chat name?

Reply:
I just reused this from `Channel` and thought someone already discovered the same question, so didn't go after it anymore.

I'd also say that it's not necessarily expected